hypothesis-python/src/hypothesis/searchstrategy/types.py

```diff
@@ -25,6 +25,7 @@
 import io
 import numbers
 import uuid
+import inspect
```

Comment:
`isort` enforces alphabetical order for imports, so `inspect` should come just before `io`.  (another [build issue](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/515071302#L803))